Sources/Sentry/SentryThreadHandle.cpp

```diff
@@ -45,6 +45,9 @@ namespace profiling {
     ThreadHandle::current() noexcept
     {
         const auto thread = pthread_mach_thread_np(pthread_self());
+        if (thread == (mach_port_t)NULL) {
+            return nullptr;
+        }
```

Comment:
I don't think we actually need this, the crash was actually happening when `current()` is dereferenced (see the callsites in this diff)